### ISSUE TITLE
Choose central languanges/technology

### ISSUE THREAD

**Issue by swoogles** [ShoulderStanders/InitialDiscussion#1]
"We write software for you" is a little too broad, let's try to narrow this down based on-

- The languages we're familiar with
- The technology we *want* to get familiar with
- What would serve potential customers (once we have some idea of who they are)

**Comment by BruceEckel** [comment]
I think there are customers out there for everything. I would like to figure out what makes me happiest to do, and then find those customers. The narrowing at our end will keep us from doing things that aren't fun.

**Comment by swoogles** [comment]
Yeah, I like starting the filtering with our interests, rather than the other way around. Let's just throw potentials on this issue and see what sticks over time.

- Kafka
- Kotlin
- Ammonite

**Comment by BruceEckel** [comment]
- Python
- Python concurrency
